contracts/StorageManager.sol

```diff
@@ -1,12 +1,15 @@
 pragma solidity 0.6.2;
+pragma experimental ABIEncoderV2;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
Actually I think here would be enough to just load the interface `IERC20.sol` right?
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/IERC20.sol

Reply:
@nduchak

Reply:
Done